src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -45,7 +38,8 @@ protected ActionResult execute() {
         gateKeeper.verifyAccessible(
                 instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
 
-        FeedbackSessionAttributes fs = extractFeedbackSessionData();
+        InstructorFeedbackAddAction newSession = new InstructorFeedbackAddAction();
+        FeedbackSessionAttributes fs = newSession.extractFeedbackSessionData();
```

Comment:
I believe what you are doing now is creating a new `InstructorFeedbackAddAction` object, then calling `extractFeedbackSessionData()` on that new object while calling `.execute()` on a `InstructorFeedbackAddAction` object. @Neurrone is right, you should be calling `extractFeedbackSessionData()` directly instead.